src/traffic_server/traffic_server.cc

```diff
@@ -1865,7 +1865,7 @@ main(int /* argc ATS_UNUSED */, const char **argv)
     initCacheControl();
     IpAllow::startup();
     ParentConfig::startup();
-    HostStatus::instance();
+    HostStatus::instance().loadStats();
```

Comment:
Should we call this function loadHostStatusFromStats() or something to indicate that this function is trying to load the Host Status from stats?